internal/repository/prisma/event.go

```diff
@@ -50,6 +50,11 @@ func (r *eventRepository) ListEvents(tenantId string, opts *repository.ListEvent
 		TenantId: *pgTenantId,
 	}
 
+	if opts.Search != nil {
+		queryParams.Search = sqlchelpers.TextFromStr(opts.Search)
+		queryParams.Search.Valid = true
```

Comment:
```suggestion
		queryParams.Search.Valid = true
```

Reply:
@abelanger5 this can be removed right :D

Reply:
Yep!